runtime/compiler/control/J9Options.cpp

```diff
@@ -2817,6 +2818,9 @@ J9::Options::unpackOptions(char *clientOptions, size_t clientOptionsSize, TR_Mem
    if (rtResolve)
       jitConfig->runtimeFlags |= J9JIT_RUNTIME_RESOLVE;
 
+   // disable caching of resolved methods if env variable TR_DisableResolvedMethodsCaching is set
+   TR_ResolvedJ9JITaaSServerMethod::_useCaching = !feGetEnv("TR_DisableResolvedMethodsCaching");
```

Comment:
TODO: add an option flag in omr which will be set at client and propagated automatically at the server. Then we can check the option directly instead of having this `useCaching` field